airmozilla/main/templates/main/home.html

```diff
@@ -11,7 +11,17 @@
 </style>
 {% endblock %}
 
-
+{% block extra_head %}
+{{ super() }}
+<meta name="keywords" content="{{ ','.join(tags) }}">
+<meta property="og:title" content="Home">
+<meta property="og:url" content="{{ url('main:home', views.home) }}">
+<meta property="og:description" content="Airmozilla Homepage') }}">
```

Comment:
Yes, there is a leftover parenthesis, I'll use that description and I'll fix that.